unischedule/README.md

```diff
@@ -1,16 +1,9 @@
 # unischedule
 
-A new Flutter project.
+Some files are automatically generated by the build process. To automatically generate these files, run the following command:
 
-## Getting Started
+```
+dart run build_runner watch
+```
 
-This project is a starting point for a Flutter application.
-
-A few resources to get you started if this is your first Flutter project:
-
-- [Lab: Write your first Flutter app](https://docs.flutter.dev/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://docs.flutter.dev/cookbook)
-
-For help getting started with Flutter development, view the
-[online documentation](https://docs.flutter.dev/), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+Code annotated with `@riverpod` or `@freezed` will be automatically generated. These files are not tracked by git.
```

Comment:
Please keep this in mind after the PR is merged to avoid any conflicts!
@iGotty @SnowArtz

Reply:
Got it! 🐱‍👤